packages/gafl-webapp-service/src/pages/order-complete/order-complete/order-complete.njk

```diff
@@ -29,7 +29,7 @@
 
             {% if (data.recurringPayment) %}
                 <h2 class="govuk-heading-m">{{ mssgs.order_complete_future_payments_title }}</h2>
-                <p class="govuk-body">{{ data.content.futurePaymentsParagraphOne }}<a class="govuk-link"  href="">{{ mssgs.order_complete_future_payments_link }}</a>{{ data.content.futurePaymentsParagraphTwo }}</p>
+                <p class="govuk-body">{{ data.content.futurePaymentsParagraphOne }}</p>
```

Comment:
In the data object, you set futurePaymentsParagraph, but here you're referring to futurePaymentsParagraphOne, so I don't think it'll work